docs/articles/utilities/lease.md

```diff
@@ -0,0 +1,105 @@
+---
+uid: lease
+title: Lease
```

Comment:
"Distributed Locks with Akka.Coordination" would be better

Reply:
Done